polkadot/xcm/src/lib.rs

```diff
@@ -428,6 +431,7 @@ pub type VersionedMultiAssets = VersionedAssets;
 #[scale_info(replace_segment("staging_xcm", "xcm"))]
 pub enum VersionedXcm<RuntimeCall> {
 	#[codec(index = 2)]
+	#[deprecated]
```

Comment:
what about other V2 types? VersionedAssets/VersionedAsset/VersionedLocation? Why just `VersionedXcm`?

Reply:
Usually that's the type that determines whether you're using v2. I can mark all the others as deprecated though.